kolibri/plugins/user/assets/src/views/FacilitySelect.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+
+  <AuthBase>
+    <div class="facility-select">
+      <KRouterLink :to="backTo" :text="coreString('goBackAction')" icon="back" />
+      <div>
+        <p>{{ label }}</p>
+        <div v-for="facility in facilityList['enabled']" :key="facility.id" class="facility-name">
+          <KButton appearance="basic-link" @click="setFacility(facility.id)">
+            {{ facility.name }}
+          </KButton>
+        </div>
+      </div>
+      <div v-if="facilityList['disabled'].length" class="disabled-facilities">
+        <p>{{ $tr('askAdminForAccountLabel') }}</p>
+        <div
+          v-for="facility in facilityList['disabled']"
+          :key="facility.id"
+          class="disabled facility-name"
```

Comment:
I don't think this `.disabled` class is needed. It should display as unstyled text even without it.